ni/actor-framework
# Resolved issue #28

Fixed the palettes in the VIP build specification.  Deleted deprecated package builds (builds 12 and 17).  Added build 18.

Here are the affected VIs:

    Send Error Report.vi
    Send Batch.vi
    Send Message and Wait for Response.vi
    Address Message.vi
    Send Self-Addressed Message.vi

Send Error Report.vi is found on the Actor Framework palette. The rest are found on the Advanced subpalette.

The only changes were to the .vipb file and the .vips.  No VIs, classes, or libraries were changed.

To test:

1.  Install build 18.
2.  Open LabVIEW, and create a blank project.
3.  Create a new VI in that project.
4.  Verify that the Actor Framework and Actor Framework » Advanced palettes contain no deprecated VIs.  Deprecated VIs are marked with a large red X superimposed on the VI icon.
5.  Add each VI from the above list to the new VI's block diagram.
6.  Inspect the dependencies list of the project, and confirm that versions of each message type (Report Error, Batch, Reply, and Self-Addressed) included in dependencies are the versions contained in Actor Framework.lvlib or AF Specialty Messages.lvlib.  (The deprecated versions of these messages are not contained in a library.)
